io/net/htx_nic_devices.py.data/htx_nic_devices.yaml

```diff
@@ -0,0 +1,9 @@
+mdt_file: 'net.mdt'
+host_ip: x.xx.xxx.xx
+peer_ip: x.xx.xxx.xy
+peer_user: "root"
+peer_password: "******"
```

Comment:
In all other tests, we have mandated passwordless ssh to be set up. Can that be followed here also please